docs/examples/dynamic_provisioning_external_gluster/README.md

```diff
@@ -0,0 +1,572 @@
+	# ssh-keygen  (take defaults)
+	# ssh-copy-id -i /root/.ssh/id_rsa.pub root@gluster25.rhs
+```
+***NOTE:*** Repeat this from each node as necessary
+
+
+#### Download and install GlusterFS.
+
+
+```
+	# yum install wget telnet -y
+	# yum install centos-release-gluster -y
+	# yum install epel-release -y
+	# yum install glusterfs-server -y
+```
+
+#### Start glusterd.
+
+```
+	# systemctl start glusterd
+	# systemctl enable glusterd
+```
+
+#### Enable firewall rules.
+
+Add Some Firewall Rules On All Nodes (if you have not disabled firewalld)
+
+```
+	# firewall-cmd --zone=public --add-service=glusterfs --permanent
+	# firewall-cmd --reload
+```
+
+
+#### Setup the Storage Pool.
+
+From one of the GlusterFS storage nodes, peer probe the other members of the pool
+
+```
+	# gluster peer probe gluster26.rhs
+	peer probe: success. 
+
+	# gluster peer status
+	Number of Peers: 1
+
+	Hostname: gluster26.rhs
+	Uuid: 6d7a3149-e163-4de8-9e63-b00e9ceacc59
+	State: Peer in Cluster (Connected)
+
+```
+
+From the other storage node (gluster26.rhs) - run the peer status
+
+```
+	# gluster peer status	
+	Number of Peers: 1
+
+	Hostname: gluster25.rhs
+	Uuid: d4cbbdbf-5dca-46ba-a7d4-2cc5d8bcdb06
+	State: Peer in Cluster (Connected)
+
+```
+
+
+
+### Installing and configuring Heketi Server Node
+
+Heketi is used to manage our gluster cluster storage (adding volumes, removing volumes, etc…). As already stated this example
+uses a stand-alone virtual machine to run Heketi and the client, but this can be installed on one of the existing Gluster Storage Nodes.
+
+
+#### Confirm and validate communication on the Heketi Server Node.
+
+- Similar to the GlusterFS storage nodes, verify /etc/hosts and firewalls
+
+```
+	# firewall-cmd --permanent --zone=public --add-port=8080/tcp
+	# firewall-cmd --reload
+```
+
+#### Install the Heketi Server.
+
+Choose an existing Gluster Storage Node or another Node that can run the Heketi Server with access to the Gluster cluster.
+For this example we are using a non-gluster stand-alone node. For more information see the [Heketi github site](https://github.com/heketi/heketi/wiki/Installation)
+
+```
+	# yum install wget telnet -y
+	# yum install centos-release-gluster -y
+	# yum install epel-release -y
+	# yum install heketi -y
+
+	# heketi version
+	Heketi 2.0.6
+	Please provide configuration file
+```
+
+
+
+
+#### Install the Heketi-Client.
+Choose an existing Gluster Storage Node or another Node that can run the Heketi Server with access to the Gluster cluster.
+Download the [latest Heketi release](https://github.com/heketi/heketi/releases) and untar it into the /etc/heketi directory.
+
+```
+	Install
+	# wget https://github.com/heketi/heketi/releases/download/v3.1.0/heketi-client-v3.1.0-HEAD.linux.amd64.tar.gz
+	# mkdir -p /etc/heketi && tar xzvf heketi-client-v3.1.0-HEAD.linux.amd64.tar.gz -C /etc/heketi
+
+	Export PATH
+	# export PATH=$PATH:/etc/heketi/heketi-client/bin
+	# heketi-cli --version
+	heketi-cli v3.1.0-HEAD
+
```

Comment:
In this doc what we have is Heketi Client v3 and Server v2 , which is not good :(.  Either we should have both v2 or v3.